WeScan/Common/QuadrilateralView.swift

```diff
@@ -64,19 +95,35 @@ final class QuadrilateralView: UIView {
     }
     
     private func commonInit() {
-        layer.addSublayer(quadLayer)
-        setupCornerButtons()
+        addSubview(quadView)
+        setupCornerViews()
+        setupConstraints()
+        quadView.layer.addSublayer(quadLayer)
+    }
+    
+    private func setupConstraints() {
+        let quadViewConstraints = [
+            quadView.leadingAnchor.constraint(equalTo: leadingAnchor),
```

Comment:
Nit: This is not in the order we once discussed: Top, left, bottom right (Basically the order of arguments in UIEdgeInsets)